modules/Poloniex.py

```diff
@@ -130,7 +130,7 @@ def _read_response(resp):
                 data = json.loads(raw_polo_response)
                 polo_error_msg = data['error']
             except:
-                if ex.code == 502:  # 502 Bad Gateway so response is likely HTML from Cloudflare
+                if ex.code == 502 or ex.code in range(520, 526, 1):  # 502 and 520-526 Bad Gateway so response is likely HTML from Cloudflare
```

Comment:
We try to keep lines to a maximum length of 120 characters, so in this situation it would be better if you moved the comment to the next line for better readability.